atomic_reactor/constants.py

```diff
@@ -62,6 +62,7 @@
 TAG_NAME_REGEX = r'^[\w][\w.-]{0,127}$'
 
 PLUGIN_KOJI_PROMOTE_PLUGIN_KEY = 'koji_promote'
+PLUGIN_KOJI_IMPORT_PLUGIN_KEY = 'koji_import'
```

Comment:
The reason for doing this is so the exit_store_metadata_in_osv3.py plugin can collect the Koji build ID and store it in the OpenShift Build annotation.

Please alter that plugin so it sets koji-build-id from whichever plugin ran, either koji_import or (failing that) koji_promote.